qutebrowser/utils/urlutils.py

```diff
@@ -443,3 +443,60 @@ def __str__(self):
             return self.msg
         else:
             return '{}: {}'.format(self.msg, self.url.errorString())
+
+
+class IncDecError(Exception):
+
+    """Exception raised by url_incdec_number on problems.
+
+    Attributes:
+        msg: The error message.
+        url: The QUrl which caused the error.
+    """
+
+    def __init__(self, msg, url):
+        super().__init__(msg)
+        self.url = url
+        self.msg = msg
+
+    def __str__(self):
+        return '{}: {}'.format(self.msg, self.url.toString())
+
+
+def url_incdec_number(url, incdec):
```

Comment:
What do you think about removing the `url_` prefix? It kinda seems redundant to me as it's in `urlutils`.